src/pages/manage_dac/ManageEditDac.jsx

```diff
@@ -250,30 +248,37 @@ export default function ManageEditDac(props) {
   return (
     isLoading ?
       <Spinner/> :
-      <div className='container container-wide'>
-        <div className='row no-margin'>
+      <div className="container container-wide">
```

Comment:
usage of " and ' is pretty inconsistent. is " the standard?

Reply:
Good catch, single quotes are the standard - not sure why the linter didn't pick this up